pennylane/transforms/qcut.py

```diff
@@ -210,3 +213,121 @@ def tape_to_graph(tape: QuantumTape) -> MultiDiGraph:
             order += 1
 
     return graph
+
+
+# pylint: disable=too-many-branches
+def contract_tensors(
+    tensors: Sequence,
+    communication_graph: MultiDiGraph,
+    prepare_nodes: Sequence[Sequence[PrepareNode]],
+    measure_nodes: Sequence[Sequence[MeasureNode]],
+    use_opt_einsum: bool = False,
+):
+    """Contract tensors according to the edges specified in the communication graph.
```

Comment:
I wasn't able to find this function in the docs, so wasn't able to check for rendering!

Reply:
Once https://github.com/PennyLaneAI/pennylane/pull/2153 is merged, I'll add a place in the main `transforms` docs :+1:

Reply:
Once https://github.com/PennyLaneAI/pennylane/pull/2153 is merged, I'll add a place in the main `transforms` docs :+1:

Reply:
Once https://github.com/PennyLaneAI/pennylane/pull/2153 is merged, I'll add a place in the main `transforms` docs :+1: